### ISSUE TITLE
Change styles of date selector modal

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#10478]
Off the back of #10234 and a chat with @j-jaworski , we decided to split that piece of work for that section as it would probably require a redesign/rethink of that area. 

![Image](https://github.com/wellcomecollection/wellcomecollection.org/assets/110461050/53c737df-0773-4619-ac8d-75dec8c55072)

Using the placeholder as a label doesn't work as it disappears once focused, but adding a label [the same way we'll do the other input fields](https://www.figma.com/file/juZQR2wi2BsYG8TDH6R4uk/Wellcome-Collection-%E2%80%93-Styles-and-Component-Library-2023?type=design&node-id=1776%3A4028&mode=dev&t=Cu6Gm9mlde1dQRlV-1) doesn't quite work either.

Feel free to move this back to the backlog, but as it's kind of part of the raised accessibility issue that was #10234 I'm starting it off in Next for now?

**Comment by j-jaworski** [comment]
I think this should be covered by the design proposal for events in search Design MVP for events in search[#10569](https://github.com/wellcomecollection/wellcomecollection.org/issues/10569)

**Link to designs in Figma**
https://www.figma.com/file/e6Nng9zRjZITtYAZHpF1dB/Events-%26-Exhibitions-in-Search?type=design&node-id=117%3A1814&mode=dev&t=y3d1d0OvliFtIqBB-1




![Image](https://github.com/wellcomecollection/wellcomecollection.org/assets/44364115/a8e2e28f-9b0e-4c41-a21d-fcc75fafa250)

**Comment by pollecuttn** [comment]
Size: small
